src/App.js

```diff
@@ -1,29 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Your components should be class-based. Kindly change this App component to be a class-based one.

Reply:
Sure let me do it right now. Thanks @TedLivist